doc/tor.md

```diff
@@ -92,13 +93,9 @@ for normal IPv4/IPv6 communication, use:
 To test your set-up, you might want to try connecting via Tor to just a
 a single Zcash hidden server.
```

Comment:
If connecting to the zcash server which has just been setup above in section 2, might want to say test and launch zcashd on a different computer.  Otherwise a user might get confused and try to connect from the same computer as the one they are trying to connect to.